pages/api/graphql.ts

```diff
@@ -132,6 +133,7 @@ const resolvers = {
                         manifestId: manifest.id,
                         label: metadataLabel,
                         value,
+                        thumbnail: manifest.thumbnail[0].id,
```

Comment:
Long term we may need to thinking about adding a conditional on this in case a thumbnail does not exist.